docs/reactivity/key-concepts-of-reactivity.md

```diff
@@ -0,0 +1,76 @@
+# Key concepts of reactivity
```

Comment:
I'm considering breaking this page down into the pages per concept. Do these concepts be correctly understood as standalone pages or they are interrelated?

Reply:
They could be seperated, but we also will have a separate section for some of those anyway. For example, I'll be creating a page for Layers.